tests/unit/IncrementButton.spec.js

```diff
@@ -1,28 +1,57 @@
+
 import {
   render, screen, fireEvent,
 } from '@testing-library/vue';
 import IncrementButton from '@/components/generics/buttons/IncrementButton.vue';
 
-test('increments value on click', async () => {
-  // The `render` method renders the component into the document.
-  // It also binds to `screen` all the available queries to interact with
-  // the component.
-  render(IncrementButton);
+describe('IncrementButton component', () => {
+  test('renders a button', async () => {
+    render(IncrementButton);
+
+    expect(screen.getByRole('button', { name: 'Increment' })).toBeTruthy();
```

Comment:
getByRole is used always for buttons and some other elements.

You can also search for the text: 
```javascript
getByText('Increment')
```

Nevertheless, searching for a role is more accurate because that also tests accessibility.